src/makeCalculator.js

```diff
@@ -37,7 +37,42 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    reset() {
+      calculator.result = 0;
+
+      return this;
+    },
+
+    add(operand) {
+      calculator.result += operand;
+    },
+
+    subtract(operand) {
+      calculator.result -= operand;
+    },
```

Comment:
Make it like a pure function.
Let this method return you a pure result, and save it in your operate method.
For example
```
add(num1, num2) {
	return num1 + num2;
}
```